tests/mock/zwave.py

```diff
@@ -0,0 +1,78 @@
+from unittest.mock import MagicMock
+
+from pydispatch import dispatcher
+
+SIGNAL_VALUE_CHANGED = 'mock_value_changed'
+SIGNAL_VALUE = 'mock_value'
+
+
+def value_changed(value):
+    """Fire a value changed."""
+    dispatcher.send(
+        SIGNAL_VALUE_CHANGED,
+        value=value,
+        node=value.node,
+        network=value.node._network
+    )
+
+    dispatcher.send(
+        SIGNAL_VALUE,
```

Comment:
Yeah, but I realize now that since we don't use it, we might as well kill it. Let me remove it.